OpenAI.SDK/ObjectModels/ResponseModels/BaseResponse.cs

```diff
@@ -13,7 +13,7 @@ public record Error
 {
     [JsonPropertyName("code")] public string? Code { get; set; }
 
-    [JsonPropertyName("message")] public string? Message { get; set; }
```

Comment:
Why do we need this? Do you have any sample responses that show why the change is necessary?

Reply:
Yes, please see the sample response in the description of commit https://github.com/betalgo/openai/pull/320/commits/80bde477c1b1c23077d5e9f20461b1ebd724f736

It came back as an array of strings (rather than a string) which caused the JSON serializer to blow up